Cargo.lock

```diff
@@ -3556,9 +3556,9 @@ dependencies = [
 
 [[package]]
 name = "parity-util-mem"
-version = "0.5.2"
+version = "0.5.1"
```

Comment:
Why the downgrade?

Reply:
It's not a downgrade compared to master, just reverted unrelated to this PR change (see REVERTME commit).